packages/ui/src/app/components/NavBar.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import { MenuIcon } from '@/common/components/icons'
+import { Colors, ZIndex } from '@/common/constants'
+import { useResponsive } from '@/common/hooks/useResponsive'
+
+import { SideBarSlider } from './SideBarSlider'
+
+export const NavBar = () => {
+  const [open, setOpen] = useState(false)
+  const { isMobile } = useResponsive()
+
+  return (
+    <>
+      <Wrapper>
+        <Link href="#">
+          <img src="./favicon.svg" width={32} height={32} alt="Pioneer Logo" />
+        </Link>
+        <MenuIconWrapper
+          onClick={() => {
+            setOpen(!open)
+          }}
+        >
+          <MenuIcon />
+        </MenuIconWrapper>
+      </Wrapper>
+      {isMobile && open && <SideBarSlider setOpen={setOpen} />}
+    </>
+  )
+}
+
+const Wrapper = styled.div`
```

Comment:
[The design](https://www.figma.com/file/EaObQEN95O0IWjSVTUQ5NN/Pioneer-RWD?type=design&node-id=706-202460&mode=dev) has a `box-shadow: 0px -1px 0px 0px rgba(187, 217, 246, 0.13) inset;` it's really subtle but I think it makes scrolling through white area better